multiversx_sdk_cli/tests/test_wallet_generator.py

```diff
@@ -37,3 +41,36 @@ def test_derive_pem_from_mnemonic(monkeypatch: Any):
         assert True
     else:
         assert False
+
+
+def test_generate_wallet_and_save_in_json_format(monkeypatch: Any):
+    output_path = Path(__file__).parent / "testdata-out" / "testWallet.json"
+
+    with monkeypatch.context() as mp:
+        mp.setattr('getpass.getpass', lambda _: "TestPassword")
+
+        result = main(['wallet', 'new', '--json', '--output-path', str(output_path)])
+    
+    assert Path.is_file(output_path) == True
+
+    if not result:
+        assert True
+    else:
+        assert False
+
+
+def test_get_account_from_json():
+    json_file = Path(__file__).parent / "testdata-out" / "testWallet.json"
+
+    try:
+        _ = Account(key_file=str(json_file), password="TestPassword")
```

Comment:
You can remove `try / catch` and do a simple assertion on `account.address.bech32() == ...`.